<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_session_window</class>
 <widget class="QMainWindow" name="create_session_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Session</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="combo_box_student_name">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Name of Student:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Name of session</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_edit_name_of_session">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_session_info">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="list_widget_member_of_session">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>741</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_wrong_inputs">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>751</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_save_session_name">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	background-color: #3474eb
}

QPushButton:enabled {
	background-color: #3474eb
	color: white;
}

QPushButton:pressed {
	background-color: #0d2f72;
	color: #fffffe;
}

QPushButton:hover:!pressed {
	background-color: E1F4FF;
	color: 0c2f70;
}

QPushButton:disabled {
	background-color: #aaaaaa;
	color: #ffffff;
}</string>
       </property>
       <property name="text">
        <string>Save name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add_student">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	background-color: #3474eb
}

QPushButton:enabled {
	background-color: #3474eb
	color: white;
}

QPushButton:pressed {
	background-color: #0d2f72;
	color: #fffffe;
}

QPushButton:hover:!pressed {
	background-color: E1F4FF;
	color: 0c2f70;
}

QPushButton:disabled {
	background-color: #aaaaaa;
	color: #ffffff;
}</string>
       </property>
       <property name="text">
        <string>Add Student</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>570</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_create_session">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	background-color: #3474eb
}

QPushButton:enabled {
	background-color: #3474eb
	color: white;
}

QPushButton:pressed {
	background-color: #0d2f72;
	color: #fffffe;
}

QPushButton:hover:!pressed {
	background-color: E1F4FF;
	color: 0c2f70;
}

QPushButton:disabled {
	background-color: #aaaaaa;
	color: #ffffff;
}</string>
       </property>
       <property name="text">
        <string>Create session</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_back">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	background-color: #3474eb
}

QPushButton:enabled {
	background-color: #3474eb
	color: white;
}

QPushButton:pressed {
	background-color: #0d2f72;
	color: #fffffe;
}

QPushButton:hover:!pressed {
	background-color: E1F4FF;
	color: 0c2f70;
}

QPushButton:disabled {
	background-color: #aaaaaa;
	color: #ffffff;
}</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
